.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+php:
+ - 7.1
+ - 7.2
```

Comment:
We might want to support PHP 5.6 ? (EOL 01.01.2019)

Or are there already some features in the code that are incompatible ?

Reply:
Except the `return type` nothing new yet.